scripts/jest/matchers/__tests__/toWarnDev-test.ts

```diff
@@ -0,0 +1,50 @@
+/* tslint:disable:no-console */
+
+describe('toWarnDev', () => {
+  describe('usage', () => {
+    test('fails with incorrect type of message', () => {
+      expect(() => {
+        // @ts-ignore:next-line
```

Comment:
why do you need this?
I would have assume you don't since you've added `no-empty: false` to tslint.json

Reply:
That's because `toWarnDev` expects a string or nothing. TypeScript complains that boolean is a wrong type.

Since we still have some tests using plain JavaScript, I figured this test can be useful. At some point when all the tests in the codebase are migrated to TypeScript, it could be removed.